Parkinson/TabBarControllers/Dashboard/APHDashboardEditViewController.m

```diff
@@ -52,11 +53,14 @@ - (void)prepareData
             APHDashboardItemType rowType = typeNumber.integerValue;
             
             switch (rowType) {
-                case kAPHDashboardItemTypeIntervalTapping:
+                case kAPHDashboardItemTypeIntervalTappingRight:
+                case kAPHDashboardItemTypeIntervalTappingLeft:
                 {
                     APCTableViewDashboardItem *item = [APCTableViewDashboardItem new];
-                    item.caption = NSLocalizedString(@"Tapping", @"");
-                    item.taskId = @"2-APHIntervalTapping-7259AC18-D711-47A6-ADBD-6CFCECDED1DF";
+                    item.caption = (rowType == kAPHDashboardItemTypeIntervalTappingRight)  ?
+                        NSLocalizedString(@"Tapping - Right", @"") :
```

Comment:
Might be a good time to take the opportunity to put in an explanatory comment for the translator.